19-employees_policy.md

```diff
@@ -23,20 +23,20 @@ Health Note is committed to ensuring all workforce members actively address secu
-   * Employees must complete this training before accessing production systems containing ePHI.
+   * Current Health Note Cybersecurity training is managed by [Mineral] (https://trustmineral.com/).
+   * Records of training are kept for all workforce members, via [Mineral] (https://trustmineral.com/).
+   * Employees **must complete** this training before accessing production systems containing ePHI.
 2. All workforce members are granted access to formal organizational policies, which include the sanction policy for security violations.
 3. The Health Note Employee Handbook clearly states the responsibilities and acceptable behavior regarding information system usage, including rules for email, Internet, mobile devices, and social media usage.
    * Workforce members are required to sign an agreement stating that they have read and will abide by all terms outlined in the Health Note Employee Handbook, along with all policies and processes described in this document.
    * A Human Resources representative will provide the agreement to new employees during their onboarding process.
 4. All workforce members are educated about the approved set of tools to be installed on workstations.
 5. All new workforce members are given HIPAA training within 30 days of beginning employment. Training includes HIPAA reporting requirements, including the ability to anonymously report security incidents, and the levels of compliance and obligations for Health Note and its Customers and Partners.
-   * Current Health Note training is hosted on Google Apps.
+   * Current Health Note training course is managed by [Mineral] (https://trustmineral.com/).
+   * Records of training are kept for all workforce members, via [Mineral] (https://trustmineral.com/).
 6. All remote (teleworking) workforce members are trained on the risks, the controls implemented, their responsibilities, and sanctions associated with violation of policies. 
-7.  All Health Note-purchased and -owned computers are to display this message at login and when the computer is unlocked: *This computer is owned by Health Note Health, Inc. By logging in, unlocking, and/or using this computer you acknowledge you have seen, and follow, these policies (https://policy.healthnote.com) and have completed this training (https://training.healthnote.com). Please contact us if you have problems with this - privacy@healthnote.com*.
-8. Employees may only use Health Note-purchased and -owned workstations for accessing production systems with access to ePHI data.
+7.  All Health Note-purchased and owned computers are to display this message at login and when the computer is unlocked: *This computer is owned by Health Note Health, Inc. By logging in, unlocking, and/or using this computer you acknowledge you have seen, and follow, these policies (https://policy.healthnote.com) and have completed HIPAA and Security training. Please contact us if you have problems with this - privacy@healthnote.com*.
```

Comment:
2 things here.

1. Do we actually have a way of setting this up because it does not occur today.
2. Do we actually have `privacy@healthnote.com`? Don't see it in google admin nor in google groups.

Reply:
@jameschu2550 to take this on as well.

it-support@healthnote.com

Reply:
Work tracked in https://healthnote.atlassian.net/browse/DEVOPS-2258.  Added custom profile to Blueprint via iMazing profile editor to enable login message and MacOS automatic updates